Sources/ProjectDescription/CopyFilesAction.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct CopyFilesAction: Equatable, Codable {
```

Comment:
I am not sure I would call it `CopyFilesAction` -> I'd still lean to `BundledFiles` as that is more descriptive of what the user wants to achieve.

Reply:
I've added a comment with more context, see if that makes sense for you... I'm leaning towards `CopyFilesPhase` or `CopyFilesAction` but I think it's important to mention Copy Files, as these are not really Bundled files...

Reply:
Yeah, bundles are used for something else in Xcode so I'd rather not use it. I like `CopyFilesAction`, we also use actions to refer to script build phases so I think it's consistent with what we've done previously.